test/distributed/pipelining/test_schedule.py

```diff
@@ -100,8 +101,51 @@ def test_ec_backward(self, ScheduleClass):
     @requires_nccl()
     @skip_but_pass_in_sandcastle_if(not TEST_MULTIGPU, "NCCL test requires 2+ GPUs")
     @parametrize("ScheduleClass", [ScheduleGPipe, Schedule1F1B])
-    def test_grad(self, ScheduleClass):
-        mod = MultiMLP(d_hid)
+    def test_schedule_with_manual(self, ScheduleClass):
```

Comment:
just curious- is there anything that test_schedule* tests that test_grad* does not test? or should we just delete test_schedule and rename test_grad to test_schedule?

Reply:
Makes sense, maybe I can consolidate them.